src/plots/linePlot.ts

```diff
@@ -14,6 +14,9 @@ export module Plots {
 
     private _autorangeSmooth = false;
 
+    // Performance options
```

Comment:
Do we have more than one?  I don't see the need for the comment here until we have more than 1?